en/datastore/reference/data-api.rst

```diff
@@ -0,0 +1,267 @@
+
+Parameters:
+  iati-identifier: String denoting an individual IATI activity ID
+
+Example API call:
+    `iati-identifier=NL-KVK-41177206-C-002350 <http://datastore.iatistandard.org/api/1/access/activity.xml?iati-identifier=NL-KVK-41177206-C-002350>`__
+
+
+recipient-country
+`````````````````
+
+Returns activities where the `recipient-country <http://iatistandard.org/activity-standard/iati-activities/iati-activity/recipient-country/>`__ element has a @code attribute value that matches your specified ISO country code.
+
+Parameters:
+  recipient-country: 2-digit ISO country code that appears on the `country codelist <http://iatistandard.org/codelists/Country/>`__
+
+Example API call:
+    `recipient-country=SS <http://datastore.iatistandard.org/api/1/access/activity.xml?recipient-country=SS>`__
+
+
+recipient-region
+````````````````
+
+Returns activities where the `recipient-region <http://iatistandard.org/activity-standard/iati-activities/iati-activity/recipient-region/>`__ element has a @code attribute value that matches your specified DAC region code.
+
+Parameters:
+    recipient-region: 3-digit DAC region code that appears on the `region codelist <http://iatistandard.org/codelists/Region/>`__
+
+Example API call:
+    `recipient-region=298 <http://datastore.iatistandard.org/api/1/access/activity.xml?recipient-region=298>`__
+
+
+reporting-org
+`````````````
+
+Returns activities where the `reporting-org <http://iatistandard.org/activity-standard/iati-activities/iati-activity/reporting-org/>`__ element has a @ref attribute value that matches your specified `Organisation identifier <http://iatistandard.org/organisation-identifiers/>`__ string.
+
+Parameters:
+    reporting-org: Organisation identifier string.
+
+Example API call:
+    `reporting-org=NL-KVK-41177206 <http://datastore.iatistandard.org/api/1/access/activity.xml?reporting-org=NL-KVK-41177206>`__
+
+
+reporting-org.type
+``````````````````
+
+Returns activities where the `reporting-org <http://iatistandard.org/activity-standard/iati-activities/iati-activity/reporting-org/>`__ element has a @type attribute value that matches your specified value.
+
+Parameters:
+    recipient-org.type: 2-digit value that appears on the `organisation type codelist<http://iatistandard.org/codelists/OrganisationType/>`__
+
+Example API call:
+    `reporting-org.type=10 <http://datastore.iatistandard.org/api/1/access/activity.xml?reporting-org.type=10>`__
+
+
+sector
+``````
+
+Returns activities where the `sector <http://iatistandard.org/codelists/Sector/>`__ element has a @code attribute value that matches your specified DAC sector code.
+
+Parameters:
+    sector: 5-digit DAC sector code that appears on the `sector codelist <http://iatistandard.org/codelists/Sector/>`__
```

Comment:
It is merely the value present in the `sector/@code` attribute and doesn't have to be a DAC-5 code.

https://discuss.iatistandard.org/t/improvements-to-the-iati-datastore-csv-query-builder-now-live/1056/5?u=hayfield

Reply:
This also impacts other elements that use vocabularies, such as `recipient-region`.